code/__DEFINES/misc.dm

```diff
@@ -378,4 +378,13 @@ var/list/bloody_footprints_cache = list()
 #define PLANT_NORMAL 0
 #define PLANT_WEED 1
 #define PLANT_MUSHROOM 2
-#define PLANT_ALIEN 3
\ No newline at end of file
+#define PLANT_ALIEN 3
+
+//Drunkenness states
+#define NOT_DRUNK 0
+#define LIGHTLY_DRUNK 1
+#define DRUNK 2
+#define HEAVILY_DRUNK 3
+#define SMASHED 4
+#define SHITFACED 5
+#define BARELY_ALIVE 6
```

Comment:
These defines are a bit too short, they could be conflicting with other defines in the future. And BARELY_ALIVE is way too generic. Use something like ALCOHOL_SOBER, ALCOHOL_TIPSY, ALCOHOL_DRUNK, it's more specific.